src/main/java/de/uni_stuttgart/it_rex/media/written/video/VideoServiceExtended.java

```diff
@@ -41,22 +43,21 @@
 import java.nio.file.Path;
 import java.security.InvalidKeyException;
 import java.security.NoSuchAlgorithmException;
-import java.util.Optional;
 
 @Service
 public class VideoServiceExtended extends VideoService {
 
-    /**
-     * One hour in seconds.
-     */
-    private static final Integer ONE_HOUR = 3600;
```

Comment:
Das hier is mir vorher garnicht aufgefallen :laughing:

Reply:
Hat doch was:D